Changelog.md

```diff
@@ -1,5 +1,91 @@
 ## Ignition Gazebo 6.x
 
+### Gazebo Sim 6.11.0 (2022-08-17)
+
+1. Add support for specifying log record period
+    * [Pull request #1636](https://github.com/gazebosim/gz-gazebo/pull/1636)
+
+1. Common widget GzColor replacement
+    * [Pull request #1530](https://github.com/gazebosim/gz-gazebo/pull/1530)
+
+1. Replace plotIcon in ComponentInspector with GzPlotIcon
+    * [Pull request #1638](https://github.com/gazebosim/gz-gazebo/pull/1638)
+
+1. Component Inspector with common widget pose plotting
+    * [Pull request #1607](https://github.com/gazebosim/gz-gazebo/pull/1607)
+
+1. Change CODEOWNERS and maintainer to Michael
+    * [Pull request #1644](https://github.com/gazebosim/gz-gazebo/pull/1644)
+
+1. Replace pose in ViewAngle with GzPose
+    * [Pull request #1641](https://github.com/gazebosim/gz-gazebo/pull/1641)
+
+1. Fix loading worlds from CLI
+    * [Pull request #1627](https://github.com/gazebosim/gz-gazebo/pull/1627)
+
+1. Add system to an entity through Component Inspector
+    * [Pull request #1549](https://github.com/gazebosim/gz-gazebo/pull/1549)
+
+1. Quick start dialog
+    * [Pull request #1536](https://github.com/gazebosim/gz-gazebo/pull/1536)
+
+1. Quiet libSDFormat console on --verbose 0
+    * [Pull request #1621](https://github.com/gazebosim/gz-gazebo/pull/1621)
+
+1. Add Ackermann Steering system (backport from Fortress)
+    * [Pull request #1613](https://github.com/gazebosim/gz-gazebo/pull/1613)
```

Comment:
This entry could be removed

Reply:
removed. [2f68bc4](https://github.com/gazebosim/gz-sim/pull/1656/commits/2f68bc4253e15d0c0a89dcde8fc21342bbcd3545)